<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>711</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="dictTab">
     <attribute name="title">
      <string>dictionary</string>
     </attribute>
     <widget class="QListView" name="indexListView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>181</width>
        <height>451</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputLineEdit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>191</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="srcLanComboBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="detLanComboBox">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="queryButton">
      <property name="geometry">
       <rect>
        <x>628</x>
        <y>30</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Query</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>283</x>
        <y>35</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>-----&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="dictTextEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>501</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="toolTab">
     <attribute name="title">
      <string>Tool | setting</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
